production/helm/loki/values.yaml

```diff
@@ -138,7 +140,7 @@ loki:
 
     ruler:
       storage:
-      {{- if or .Values.minio.enabled (eq .Values.loki.storage.type "s3") (eq .Values.loki.storage.type "gcs") }}
+      {{- if or .Values.minio.enabled (eq .Values.loki.storage.type "s3") (eq .Values.loki.storage.type "gcs") (eq .Values.loki.storage.type "azure") }}
```

Comment:
probably getting time to move this into a template, but we can do that in a later PR.

Reply:
please, later PR.